usr/src/cmd/cmd-inet/usr.sbin/snoop/snoop_ntp.c

```diff
@@ -210,7 +210,7 @@ interpret_ntp(int flags, struct ntpdata *ntp_pkt, int fraglen)
 				 */
 				break;
 		    }
-		    (void) sprintf(get_line((char *)ntp->keyid -
+		    (void) sprintf(get_line((char *)(uintptr_t)ntp->keyid -
```

Comment:
Is this correct despite `keyid` being a `uint32_t`?

Reply:
Who knows. I tested snoop but don't have encrypted ntp so this code doesn't get executed. I added the last bits with things that were up for review before so may look familiar. Only real new thing is nohup which needs definition of some registers which I think I deducted right from looking at the amd/sparc coding. This is also probably where it ends for me I will start running arm based versions the moment we put the cpu into idle mode currently its burning cpu on native hardware and as such burning power.

Reply:
I'm running the images Andy makes under proxmox on a Rpi4 with 8 Gb of RAM. So its running on a native cpu and as
such is way faster then when running it emulated under qemu. So its essentially running under kvm on PiOS. I have to
limit things to 1 cpu as it fails to online extra cpu's. I have multiple setups 1 8 Gb one running "production" and one 8 Gb
one for testing all kinds of arm builds of all kind of operating systems. And a 2 Gb one but that is not really usable to
run any Vms,. What I do see is that its burning all of the 1 cpu cores assigned to it.

Reply:
https://github.com/richlowe/illumos-gate/issues/18